catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -126,7 +136,30 @@ private CqlQueryResponse executeCqlQuery(CqlRequest cqlRequest)
                 filterBuilder);
 
         Stopwatch stopwatch = Stopwatch.createStarted();
-        QueryResponse response = catalogFramework.query(request);
+        QueryResponse response = null;
+
+        if (configuration.isDisableLocalCatalog()) {
+            Set<String> sourceIds;
+            if (CollectionUtils.isNotEmpty(request.getSourceIds())) {
+                sourceIds = new HashSet<>(request.getSourceIds());
+            } else {
+                sourceIds = new HashSet<>(catalogFramework.getSourceIds());
+            }
+            sourceIds.remove(catalogFramework.getId());
+            request = new QueryRequestImpl(request.getQuery(),
+                    false,
+                    sourceIds,
+                    request.getProperties());
+
+            if (CollectionUtils.isNotEmpty(sourceIds)) {
```

Comment:
I just mean the existing behavior of the catalog says if there are no source ids and its not enterprise we do have a default,
https://github.com/codice/ddf/blob/master/catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/QueryOperations.java#L772-L777

and the ui currently does allow you to send a query with no sources